src/main/java/io/stargate/sgv2/jsonapi/api/model/command/builders/FilterClauseBuilder.java

```diff
@@ -199,17 +210,16 @@ private List<ComparisonExpression> createComparisonExpressionList(
         return comparisonExpressionList;
       }
 
-      // Before validating Filter path, check for special cases:
```

Comment:
No longer needed as `validateFilterClausePath` no longer reports error for these well-known special fields.